AdyenActions/AdyenActionComponent.swift

```diff
@@ -276,7 +278,19 @@ public final class AdyenActionComponent: ActionComponent, ActionHandlingComponen
         component.handle(action)
         currentActionComponent = component
     }
-    
+
+    private func handleRedirectableAwait(_ action: RedirectableAwaitAction) {
```

Comment:
This this can be named similar to others as `handle(_ action: redirectableAwaitAction)`